docs/_data/repoStatus.csv

```diff
@@ -1,74 +1,85 @@
-Button - close,x,,,,
-Skip to main content,,,,,
-Tabs - Open,x,,,,
-Tabs - Box,x,,,,
-Search bar,,,,,
-Search combo,,,,,
-Footer group,,,,,
-Accordion,,x,x,,
-Filter box,,x,,,
-Form - full,,x,,,
-Form - input group - money,,,,,
-Form - input group - email,,,,,
-Form - input group - username,,,,,
-Form - input group - date,,,,,
-Form - input group - search,,,,,
-Form - input group - help,,,,,
-Alert - inline,x,,,,
-Alert - inline variation,x,,,,
-Alert - toast,x,,,,
-Navigation,,x,,,
-Navigation Secondary,x,,,,
-Cookie banner,,,,,
-Logo bar,,,x,,
-Announcement bar,,,x,x,
-Dialog,x,,,,
-Video with thumbnail,,,x,,
-Quote,x,x,x,,
-Jump links - horizontal,,,,,
-Jump links - vertical,,,,,
-Fixed bottom banner,,,x,x,
-Audio player,,,x,,
-Tooltip,x,,,,
-Promo band - full,,x,,,
-Promo band - skinny,,x,,,
-Card - basic,,x,,,
-Card - lists,,x,,,
-Card - data/fast facts,,,x,,
-Card - logo,,x,,,
-Card - bar/with hat,,x,,,
-Card - icon,,x,,,
-Card - image,,x,,,
-Card - asset,,,x,,
-Card - quote,,x,,,
-Card - avatars,,,x,,
-Card - video,,,x,,
-Card - pricing,,,x,,
-Card - logo slider,,,x,,
-Card - name slider,,,x,,
-Avatar,x,,,,
-Tag - Red - filled and unfilled,x,,,,
-Tag - Orange - filled and unfilled,x,,,,
-Tag - Green - filled and unfilled,x,,,,
-Tag - Cyan - filled and unfilled,x,,,,
-Tag - Blue - filled and unfilled,x,,,,
-Tag - Purple - filled and unfilled,x,,,,
-Tag - Gray - filled and unfilled,x,,,,
+id,Element/Pattern,RHDS,WebRH,WebDMS,Adobe Target,
+Accordion,Small size,x,x,,,
+Accordion,Large size,x,x,,,
+Alert,Inline,x,,,,
+Alert,Inline variation,x,,,,
+Alert,Toast,x,,,,
+Avatar,Avatar,x,,,,
+Badge,Badge,x,,,,
+Blockquote,Default size,x,,,,
+Blockquote,Large size,x,,,,
+Button,Danger,x,,,,
+Button,Primary,x,,,,
+Button,Secondary,x,,,,
+Button,Tertiary,x,,,,
+Button,Link,x,,,,
+Button,Play,x,,,,
+Button,Close,x,,,,
+Card,Basic,,x,,,
+Card,Lists,,x,,,
+Card,Data/fast facts,,,x,,
+Card,Logo,,x,,,
+Card,Bar/with hat,,x,,,
+Card,Icon,,x,,,
+Card,Image,,x,,,
+Card,Asset,,,x,,
+Card,Quote,,x,,,
+Card,Avatars,,,x,,
+Card,Video,,,x,,
+Card,Pricing,,,x,,
+Card,Logo slider,,,x,,
+Card,Name slider,,,x,,
+Cta,Primary,x,,,,
+Cta,"Primary,video",x,,,,
+Cta,Secondary,x,,,,
+Cta,"Secondary,video",,,,,
+Cta,Brick,x,,,,
+Cta,"Brick,icon",x,,,,
+Cta,Default,x,,,,
+Cta,"Default,video",x,,,,
+Dialog,Dialog,x,,,,
+Footer,Footer,x,,,,
+Navigation Secondary,Secondary navigation,x,,,,
+Pagination,Compact size,x,,,,
+Pagination,Full size,x,,,,
+Promo band, Full,,x,,,
+Promo band, Skinny,,x,,,
+Spinner,Small size,x,,,,
+Spinner,Medium size,x,,,,
+Spinner,Large size,x,,,,
+Stat,Default size,x,,,,
+Stat,Large size,x,,,,
+Tabs,Open,x,,,,
+Tabs,Boxed,x,,,,
+Tag,Red - filled and unfilled,x,,,,
+Tag,Orange - filled and unfilled,x,,,,
+Tag,Green - filled and unfilled,x,,,,
+Tag,Cyan - filled and unfilled,x,,,,
+Tag,Blue - filled and unfilled,x,,,,
+Tag,Purple - filled and unfilled,x,,,,
+Tag,Gray - filled and unfilled,x,,,,
+Tooltip,Tooltip,x,,,,
+,12-column grid,,x,x,,
```

Comment:
@coreyvickery I believe lines 62-85 is data for the repo status tables that we should be displaying under Patterns. Do you want to break this out into a separate issue for us to clean up?

Reply:
@eyevana Yes please.

Reply:
Thank you!